### DISCUSSION TITLE
how I fix this?

### DISCUSSION THREAD

**Discussion by yuchen1204** [coder/code-server#5899] [Q&A]
![image](https://user-images.githubusercontent.com/92127505/208156250-c6513506-af16-48fb-b115-6da817530005.png)

**Comment by jsjoeio** [comment]
Oh your node version is too high. Can you try re-running with the flag `--ignore-engines`?

**Reply by yuchen1204** [reply]
Which version of node should I use

**Reply by jsjoeio** [reply]
16!

**Reply by yuchen1204** [reply]
![image](https://user-images.githubusercontent.com/92127505/208160622-ae6cf0b9-0b45-487b-a7cf-b1379efcafec.png)

**Reply by yuchen1204** [reply]
what should i do next

**Reply by jsjoeio** [reply]
Hmm looks like you installed it to your `frontend` directory? Maybe try `node ./node_modules/code-server/bin --port 3000`
